### PR TITLE
Update group detail when members join or leave

### PR DESCRIPTION
**Pull request by hv0905**
Update group detail when members join or leave

### CODE REVIEW

**Review by Anduin2017 — APPROVED** [review]

**Review by xxyzz — APPROVED** [review]

File: src/app/Services/MessageService.ts [hunk 1]
```diff
@@ -97,6 +98,26 @@ export class MessageService {
                         this.timerService.destructTime, 'info');
                 }
                 break;
+            case EventType.NewMemberEvent:
+                evt = ev as NewMemberEvent;
+                if (this.conversation && this.conversation.id === evt.conversationId) {
+                    this.conversationApiService.ConversationDetail(evt.conversationId)
```
Comment (Anduin2017) [inline src/app/Services/MessageService.ts:104]: I guess that you don't have to call API again. For Stargate already gave everything you need.

Comment (hv0905) [reply src/app/Services/MessageService.ts:104]: I wanted to do this. However, Stargate just sent `KahlaUser`, but not `UserGroupRelation`

Comment (Anduin2017) [reply src/app/Services/MessageService.ts:104]: Because when a user leaves, the `UserGroupRelation` instance does not exist anymore.

Comment (hv0905) [reply src/app/Services/MessageService.ts:104]: Maybe we should make NewMemberEvent return `UserGroupRelation`, in this way, we don't have to call `ConversationDetail` api again.

**Pull request merged**